versioned_docs/version-3.x/samples.md

```
@@ -18,3 +18,9 @@ A simple TypeScript application that demonstrates how to model a minimum blog ap
 https://github.com/zenstackhq/v3-sample-todo-nextjs
 
 A multi-tenant Todo app built with Next.js, Auth.js, TanStack Query, and ZenStack v3.
+
+## Rallly
+
+https://github.com/zenstackhq/rallly
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)